Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -29,7 +27,6 @@ public void RoslynPlugin_GenerateForValidAnalyzer_Succeeds()
             // Arrange
             TestLogger logger = new TestLogger();
             string outputDir = TestUtils.CreateTestDirectory(this.TestContext, ".out");
-            PluginInspector inspector = CreatePluginInspector(logger);
```

Comment:
Fixed up this test to not use the JDK.
Previously, we built and executed the Java PluginInspector class, which examined the new .jar, created the extensions, and dumped data about them to an XML now.
Now, we just check that the jar contains the expected XML files, and that those files have the expected content.